packages/module/patternfly-docs/content/extensions/chatbot/examples/Messages/MessageWithSources.tsx

```diff
@@ -10,6 +10,35 @@ export const MessageWithSourcesExample: React.FunctionComponent = () => {
 
   return (
     <>
+      <Message
```

Comment:
Can we move this to be the second example, beneath the "Example with sources" one?

This example is also starting to get long, which I think is fine for now, but could be another opportunity to use the radios or checkboxes for toggling some of the features